redirects/16.0.txt

```diff
@@ -55,6 +55,7 @@ applications/sales/subscriptions/sales_flow/closing.rst applications/sales/subsc
 applications/sales/subscriptions/sales_flow/create_a_quotation.rst applications/sales/subscriptions.rst                               # subscriptions/sales_flow/create_a_quotation --> subscriptions
 applications/sales/subscriptions/sales_flow/renewals.rst applications/sales/subscriptions/renewals.rst                                # sales_flow/* --> *
 applications/sales/subscriptions/sales_flow/upselling.rst applications/sales/subscriptions/upselling.rst                              # sales_flow/* --> *
+applications/sales/crm/performance/google_spreadsheets.rst applications/finance/spreadsheet/templates.rst                             # removed crm doc and directed to finance doc
```

Comment:
Small change to follow the existing comment formatting, and lets move to the top of this section to maintain alphabetical order 🙂
```suggestion
applications/sales/crm/performance/google_spreadsheets.rst applications/finance/spreadsheet/templates.rst                             # /google_spreadsheets --> /spreadsheet/templates
```